### PR TITLE
Add a Reason Why AS3 is Currently not Supported

### PR DESCRIPTION
**Pull request by TheTort**
This Would Make it so People Would Understand why AS3 Is not supported currently. And Not Ask The Ruffle's Discord When is As3 Support Coming.

### CODE REVIEW

**Comment by Herschel** [comment]
Thank you, and welcome!
I like the sentiment, but the reason for the original message was because most users won't know what "ActionScript 3" is, so we want to avoid loading the message with these terms. The error message also links to the FAQ which answers this thoroughly, so I think this could be more appropriate in the FAQ.

**Comment by TheTort** [comment]
Hmmm Ok! I'll Try Something Different. Thanks Herschel!

**Comment by relrelb** [comment]
@TheTort What's the status of this PR? Is it pending future changes? Is it abandoned and can be closed?

**Comment by TheTort** [comment]
Changed It to Tell To Join the Discord and Read the FAQ

File: web/packages/core/src/ruffle-player.ts [hunk 1]
```diff
@@ -1197,7 +1197,7 @@ export class RufflePlayer extends HTMLElement {
         // TODO: Pause content until message is dismissed
         div.innerHTML = `<div class="message">
             <p>Flash Player has been removed from browsers in 2021.</p>
-            <p>This content is not yet supported by the Ruffle emulator and will likely not run as intended.</p>
```
Comment (relrelb) [inline web/packages/core/src/ruffle-player.ts:1200]: Why removing this line?

File: web/packages/core/src/ruffle-player.ts [hunk 2]
```diff
@@ -1197,7 +1197,7 @@ export class RufflePlayer extends HTMLElement {
...
+            <p>For more info check the #faq in <a href="discord.gg/ruffle>The Ruffle Discord </a></p>
```
Comment (relrelb) [inline web/packages/core/src/ruffle-player.ts:1200]: Please strip the trailing space before `</a>`.

**Review by relrelb — CHANGES_REQUESTED** [review]

File: web/packages/core/src/ruffle-player.ts (hunk 2 above)
Comment (TheTort) [reply web/packages/core/src/ruffle-player.ts:1200]: oops i'll fix it right now

**Comment by TheTort** [comment]
Hopefully this Fixes It

File: web/packages/core/src/ruffle-player.ts [hunk 3]
```diff
@@ -852,7 +852,7 @@ export class RufflePlayer extends HTMLElement {
             }
 
             for (const node of Array.from(elem.children)) {
-                this.appendChild(node);
+                this.appendChild(node);F
```
Comment (relrelb) [inline web/packages/core/src/ruffle-player.ts:855]: Please fix this typo.

Comment (TheTort) [reply web/packages/core/src/ruffle-player.ts]: fixed

**Comment by relrelb** [comment]
Wait, currently the message already links to the FAQ in the wiki (https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users), so I don't really get the point of this PR.

**Comment by TheTort** [comment]
Oh! I'll Close It Then.

**Pull request closed without merging**